packages/apps/tools/src/pages/faucet/existing/index.tsx

```diff
@@ -136,6 +140,16 @@ const ExistingAccountFaucetPage: FC = () => {
           }}
           body={requestStatus.message}
         />
+        {showNotification ? (
+            <FormStatusNotification
+              status={'erroneous'}
+              statusBodies={{
+                erroneous: t('You cannot use faucet on Mainnet.'),
```

Comment:
I'm not sure whether this is the appropriate message. Maybe it should be more of a warning, indicating to users that the Faucet will only fund users on testnet, therefor switching between networks has no effect on this call. Not sure what Isa or Albert thinks about it?